### PR TITLE
Show observation config errors in GUI

### PR DESCRIPTION
**Pull request by eivindjahren**
Propagate error messages from observation parsing to suggester gui

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Updated documentation
- [x] Ensured new behaviour is tested

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Review by oyvindeide — APPROVED** [review]
LGTM!

File: src/clib/lib/enkf/enkf_obs.cpp [hunk 1]
```diff
@@ -1017,35 +1006,42 @@ py::handle pybind_alloc(int history_,
...
+                                                "enkf_conf", config_file),
+                  conf_instance_free);
 
-    const char *errors = conf_instance_get_path_error(enkf_conf);
+    const char *errors = conf_instance_get_path_error(enkf_conf.get());
     if (errors) {
-        enkf_obs_reinterpret_DT_FILE(errors);
-        exit(1); // No need to free errors...
+        throw exc::invalid_argument{
+            "The following keywords in your configuration did not resolve to a "
```
Comment (oyvindeide) [inline src/clib/lib/enkf/enkf_obs.cpp:1019]: Can this happen? Is there test coverage for this?

Comment (eivindjahren) [reply src/clib/lib/enkf/enkf_obs.cpp:1019]: I changed the test so that it should be clear what is going on :)

**Review by oyvindeide — APPROVED** [review]

**Comment by eivindjahren** [comment]
test everest please

**Comment by eivindjahren** [comment]
jenkins test everest please

**Pull request merged**